src/algorithms/expected_consensus.md

```diff
@@ -292,10 +339,16 @@ w[r+1] = w[r] + floor(1000 * (wForkFactor(wPowerFactor[r+1] + wBlocksFactor[r+1]
 
 with, for a given tipset ts in round r+1:
 
-- wBlocksFactor[r+1] = v * |blocksInTipset(ts)|
+- wBlocksFactor[r+1] = v * |blocksInTipset(ts)| = v * k
   - with v = vt * log2(totalPowerAtTipset(ts) * vs)/vs
```

Comment:
```
v = vt * log2(totalPowerAtTipset(ts) * vs) / vs
v = vt * (log2(totalPowerAtTipset(ts) + log2(vs)) / vs
v = vt * log2(totalPowerAtTipset(ts) / vs + vt * log2(vs) / vs
v = vt * log2(totalPowerAtTipset(ts) / vs + sigma; where sigma = vt * log2(vs) / vs
```

Now what I don't understand is:
given parameters below:
```
vt = 350
vs = 1e-5
sigma = -581337416
```

Unless `log2` is already used as the proposed by me `log2b(x)`.

Reply:
Proposed change:
redefine `vs`:  as `vS = 1/vs` - fractional `vs` hides divisions then:
`v = vt * vS * log2b(totalPowerAtTipset(ts)/vS)`